prestoadmin/server.py

```diff
@@ -80,11 +84,254 @@ def new_sysnode_processor(node_info_rows):
 CONNECTOR_INFO_SQL = 'select catalog_name from system.metadata.catalogs'
 _LOGGER = logging.getLogger(__name__)
 
+DOWNLOAD_DIRECTORY = '/tmp'
+DEFAULT_RPM_NAME = 'presto-server-rpm.rpm'
+LATEST_RPM_URL = 'https://repository.sonatype.org/service/local/artifact/maven' \
+                 '/content?r=central-proxy&g=com.facebook.presto' \
+                 '&a=presto-server-rpm&e=rpm&v=RELEASE'
+
+
+class LocalPrestoRpmFinder:
+    def __init__(self, local_path):
+        self.local_path = local_path
+
+    @staticmethod
+    def _check_rpm_uncorrupted(rpm_path):
+        old_stderr = sys.stderr
+        sys.stderr = open(os.devnull, 'w')
+        try:
+            package.check_if_valid_rpm(rpm_path)
+        except SystemExit as system_exit:
+            if 'Corrupted RPM' in system_exit.message:
+                print('Removing corrupted rpm at: %s' % rpm_path)
```

Comment:
This should probably use fabric's warn() function. It should probably also say something like "Trying to remove ..."